src/main/java/com/boot/user/service/EmailService.java

```diff
@@ -22,14 +22,15 @@
 
 @Slf4j
 @Service
-@AllArgsConstructor
+@RequiredArgsConstructor
 public class EmailService {
 
     private static final String CONFIRMATION_EMAIL_TEMPLATE = "/templates/email-template.vm";
 
     private static final String RESET_PASSWORD_EMAIL_TEMPLATE = "/templates/reset-password-email-template.vm";
 
-    AppConfig appConfig;
+    @Value("${user.service.url}")
```

Comment:
isn't there any possibility to use in this case feign option aswell?

Reply:
using feign for this is not possible. we could point it to the gateway url